### PR TITLE
Processing for Release.

### PR DESCRIPTION
**Pull request by cs-shivaji-kolape**
Description
-----
- Added following changes
  1. Reformatted code.
  2. Updated connector Logo
  3. Added Help Doc Link and Updated description as per help Doc

UTC
-----
- Installed the Connector using tgz file.
- Verified updated changes(logo, help doc link and descriptions) reflected on latest TGZ.

### CODE REVIEW

File: oracle-db/info.json [hunk 1]
```diff
@@ -1,84 +1,85 @@
...
+  "category": "utilities",
+  "help_online": "https://docs.fortinet.com/document/fortisoar/1.0.0/oracle-database/463/oracle-database-v1-0-0",
+  "configuration": {
+    "fields": [
+      {
+        "name": "host",
+        "required": true,
+        "type": "text",
+        "visible": true,
+        "title": "Host",
```
Comment (cs-renuka-andhale) [inline oracle-db/info.json:18]: Make it "Server URL"

File: oracle-db/info.json [hunk 2]
```diff
@@ -1,84 +1,85 @@
...
+    ]
+  },
+  "operations": [
+    {
+      "operation": "db_query",
+      "annotation": "db_query",
+      "title": "Query DB",
+      "description": "Performs a query on the configured database based on the query string you have specified.",
+      "enabled": true,
+      "output_schema": {},
```
Comment (cs-renuka-andhale) [inline oracle-db/info.json:67]: Output Schema is missing

File: oracle-db/info.json [hunk 3]
```diff
@@ -1,84 +1,85 @@
...
-                }
-            ]
+          "name": "query_string",
+          "value": "",
+          "required": true,
+          "type": "text",
+          "visible": true,
+          "title": "Query String",
+          "editable": true,
+          "description": "Specify the query that you want to run on the configured database from which you want to retrieve the required data."
```
Comment (cs-renuka-andhale) [inline oracle-db/info.json]: Please provide example for query also in tooltip.

File: oracle-db/info.json [hunk 4]
```diff
@@ -1,84 +1,85 @@
...
-    "cs_approved": false,
-    "cs_compatible": true,
-    "help_online": ""
+      ]
+    }
+  ],
+  "cs_approved": false,
+  "cs_compatible": true
 }
+
```
Comment (cs-renuka-andhale) [inline oracle-db/info.json:87]: In playbooks add description, Trigger Button Label.

File: oracle-db/info.json (hunk 2 above)
Comment (cs-shivaji-kolape) [reply oracle-db/info.json:67]: This action execute DB query and DB query response not static. its depends on query. query like (insert, delete, select, update etc.)

File: oracle-db/info.json (hunk 1 above)
Comment (cs-shivaji-kolape) [reply oracle-db/info.json:18]: Host also the correct name.

**Review by cs-renuka-andhale — APPROVED** [review]

**Pull request merged**